### DISCUSSION TITLE
Send Hotkey to specific client

### DISCUSSION THREAD

**Discussion by DrDronez** [debauchee/barrier#1901] [Ideas]
Just a simple feature request, Sending a hotkey combo to a specific client. My particular use case is: one of my client PC's is where I usually run Discord. I'd like to be able to send my Push-To-Talk hotkey to it without losing focus on whatever server/client other machine I'm actively working on. I'd imagine there are other use cases, but not sure how many would find it useful.

**Comment by thenewlebowski** [comment]
This behavior already exists. If you create a new hot key and configure its action to `press and release a hotkey` with the `only on these screens` selected with the value set as your client that is running your discord application. I was able to make `Control+Shift+d` on my host map to `Control+Shift+F1` on my client. Of course you will have to add the new hot keys to discord.

**Comment by ghost** [comment]
Related question:
I am pressing 'x' on the host machine to mimic pressing 'y' on the client machine. This is working fine. However no 'x' appears in chat on the host machine. Just the 'y' appears on the client machine. How can I make 'y' appear on the client AND ALSO 'x' appear on the host. I have tried various of hotkey rules to no avail.
